src/views/Conference.vue

```diff
@@ -17,15 +18,25 @@ export default {
     Page,
   },
   props: {
-    room: { type: String, required: true },
+    room: { type: String, default: '' },
   },
-  mounted() {
+  async mounted() {
+    const connection = await BrowserWindowMessageConnection({
+      origin: 'https://localhost:8080',
+    });
+
+
+    connection.connect(({ room }) => {
+      console.log({ room });
+      this.$router.push({ name: 'conference', params: { room } });
+    });
+
     // eslint-disable-next-line no-new
-    new JitsiMeetExternalAPI('test.league.aeternity.org', {
+    new JitsiMeetExternalAPI('localhost:8080 ', {
```

Comment:
what domain should I use? test.league.aeternity.org or league.aeternity.org?

Reply:
you may extract it to https://github.com/aeternity/superhero-ui/blob/develop/.env and to use league.aeternity.org by default

Reply:
Thanks